sdk/nodejs/runtime/rpc.ts

```diff
@@ -590,16 +595,21 @@ export function unwrapRpcSecret(obj: any): any {
 }
 
 /** @internal */
-export function containsUnknownValues(value: unknown): boolean {
-    if (value === unknownValue) {
-        return true;
-    } else if (
+export function isPrimitive(value: unknown): boolean {
```

Comment:
Does this need to be exported? It didn't look like it's used externally. If not, remove `export` keywords and `@internal`.

Reply:
Sounds good!